### ISSUE TITLE
Wiki page

### ISSUE THREAD

**Issue by jinywu** [martin-majlis/Wikipedia-API#18]

**Comment by jinywu** [comment]
The property text is empty when the wiki page doesn't contains any "==" .
For example, https://ja.wikipedia.org/wiki/%E7%9F%A5%E8%A6%9A%E5%BF%83%E7%90%86%E5%AD%A6

**Comment by jinywu** [comment]
Reproduc code: 
import wikipediaapi
wiki_wiki = wikipediaapi.Wikipedia(language='ja', extract_format=wikipediapai.ExtractFormat.WIKI)
p_wiki=wiki_wiki.page('知覚心理学')
print(p_wiki.text)

**Comment by jinywu** [comment]
Code:
import wikipediaapi
wiki_wiki = wikipediaapi.Wikipedia(language='ja', extract_format=wikipediapai.ExtractFormat.WIKI)
p_wiki=wiki_wiki.page('知覚心理学')
print(p_wiki.text)

**Comment by jinywu** [comment]
The property text is empty when the wiki page doesn't contain any "==" .
For example, https://ja.wikipedia.org/wiki/%E7%9F%A5%E8%A6%9A%E5%BF%83%E7%90%86%E5%AD%A6

**Comment by martin-majlis** [comment]
@jinywu : This issue is solved in release 0.4.3. Thanks for reporting this bug.

```
import wikipediaapi
wiki_wiki = wikipediaapi.Wikipedia(language='ja', extract_format=wikipediaapi.ExtractFormat.WIKI)
p_wiki=wiki_wiki.page('知覚心理学')
print(p_wiki.summary)
print(p_wiki.text)

wiki_html = wikipediaapi.Wikipedia(language='ja', extract_format=wikipediaapi.ExtractFormat.HTML)
p_html=wiki_wiki.page('知覚心理学')
print(p_html.summary)
print(p_html.text)
```
